config.toml.example

```diff
@@ -29,6 +29,10 @@
 	mainFacebook = "URY1350"
 	newsTwitter = "URYNews"
 
+	[pageContext.sustainer]
+		name = "Overnight Owen"
```

Comment:
Currently the deploy script just copies the example file, so it'd certainly make it easier to deploy, but it would force us to actually do something properly...